docs/tutorials/06_torch_runtime.ipynb

```diff
@@ -0,0 +1,1327 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate random dataset\n",
+    "\n",
+    "# Select dataset dimension (num_inputs) and size (num_samples)\n",
+    "num_inputs = 2\n",
+    "num_samples = 20\n",
+    "\n",
+    "# Generate random input coordinates (X) and binary labels (y)\n",
+    "X = 2*algorithm_globals.random.random([num_samples, num_inputs]) - 1\n",
+    "y01 = 1*(np.sum(X, axis=1) >= 0)  # in { 0,  1}, y01 will be used for CircuitQNN example\n",
+    "y = 2*y01-1                       # in {-1, +1}, y will be used for OplowQNN example\n",
+    "\n",
+    "# Convert to torch Tensors\n",
+    "X_ = Tensor(X)\n",
+    "y01_ = Tensor(y01).reshape(len(y)).long()\n",
+    "y_ = Tensor(y).reshape(len(y), 1)\n",
+    "\n",
+    "# Plot dataset\n",
+    "for x, y_target in zip(X, y):\n",
+    "    if y_target == 1:\n",
+    "        plt.plot(x[0], x[1], 'bo')\n",
+    "    else:\n",
+    "        plt.plot(x[0], x[1], 'go')\n",
+    "plt.plot([-1, 1], [1, -1], '--', color='black')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 44,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create custom torch dataset class\n",
+    "class TorchDataset(Dataset):\n",
+    "\n",
+    "    \"\"\"Map-style dataset\"\"\"\n",
```

Comment:
As I claimed above, typically the newline is below a class's docstring, rather than above it.